homeassistant/components/apache_kafka/manifest.json

```diff
@@ -2,6 +2,7 @@
   "domain": "apache_kafka",
   "name": "Apache Kafka",
   "codeowners": ["@bachya"],
+  "disabled": "Dependencies does not support Python 3.12",
```

Comment:
We cannot merge this in this state. This will disable them even before we officially support 3.12.

Reply:
I suggest we, for now, remove them at wheels building time only; until it matters (when we start shipping on 3.12).